nipype/interfaces/base/support.py

```diff
@@ -235,14 +238,166 @@ def version(self):
         return self._version
 
 
-def load_template(name):
+def format_help(cls):
     """
-    Deprecated stub for backwards compatibility,
-    please use nipype.interfaces.fsl.model.load_template
+    Prints help text of a Nipype interface
+
+    >>> from nipype.interfaces.afni import GCOR
+    >>> GCOR.help()  # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
+    Wraps the executable command ``@compute_gcor``.
+    <BLANKLINE>
+    Computes the average correlation between every voxel
+    and ever other voxel, over any give mask.
+    <BLANKLINE>
+    <BLANKLINE>
+    For complete details, ...
 
     """
-    from ..fsl.model import load_template
-    iflogger.warning(
-        'Deprecated in 1.0.0, and will be removed in 1.1.0, '
-        'please use nipype.interfaces.fsl.model.load_template instead.')
-    return load_template(name)
+    from ...utils.misc import trim
+
+    docstring = []
+    cmd = getattr(cls, '_cmd', None)
+    if cmd:
+        docstring += ['Wraps the executable command ``%s``.' % cmd, '']
+
+    if cls.__doc__:
+        docstring += trim(cls.__doc__).split('\n') + ['']
+
+    allhelp = '\n'.join(
+        docstring +
+        _inputs_help(cls) + [''] +
+        _outputs_help(cls) + ['']  +
+        _refs_help(cls)
+    )
+    return allhelp.expandtabs(4)
```

Comment:
I think this will restore the indentation:

```suggestion
    return allhelp.expandtabs(8)
```